packages/page-staking/src/Nomination/StahesTable.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Table } from '@polkadot/react-components';
+import NominatedAccount from '@polkadot/app-staking/Nomination/NominatedAccount';
+import { useApi, useCall } from '@polkadot/react-hooks';
+import { useTranslation } from '@polkadot/app-staking/translate';
+import { ActiveEraInfo, EraIndex } from '@polkadot/types/interfaces';
```

Comment:
types always first, <blank line>, external imports, <blank line>, local imports. (Aligning with what is done elsewhere through the whole GH org)